### PR TITLE
Add number of total model parameters to UserData.

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: models/model_dirac/model_dirac_deltax.cpp [hunk 1]
```diff
@@ -19,12 +19,6 @@ status = w_model_dirac(t,x,NULL,user_data);
 
               } break;
```
Comment (FFroehlich) [inline models/model_dirac/model_dirac_deltax.cpp]: did you remove the event for compilation?

File: @amimodel/generateC.m [hunk 2]
```diff
@@ -347,7 +347,7 @@ function generateC(this)
 fprintf(fid,'#include <include/udata_accessors.h>\n');
 fprintf(fid,'                \n');
 fprintf(fid,'                void init_modeldims(UserData *udata){\n');
-fprintf(fid,['                   npmodel = ' num2str(this.np) ';\n']);
+fprintf(fid,['                   nplist = ' num2str(this.np) ';\n']);
```
Comment (FFroehlich) [inline @amimodel/generateC.m]: this should be np and not nplist!

Comment (dweindl) [inline @amimodel/generateC.m]: Correct. Will fix.

File: models/model_dirac/model_dirac_deltax.cpp (hunk 1 above)
Comment (dweindl) [inline models/model_dirac/model_dirac_deltax.cpp]: I did not change anything in the model. This is what I got out of running the first 3 lines of the example_dirac.

File: @amimodel/generateC.m (hunk 2 above)
Comment (dweindl) [inline @amimodel/generateC.m]: Ah, you did already. Thx.

File: models/model_dirac/model_dirac_deltax.cpp (hunk 1 above)
Comment (FFroehlich) [inline models/model_dirac/model_dirac_deltax.cpp]: okay this should be fixed now

**Pull request merged**